java/flight/flight-sql-jdbc-core/src/main/java/org/apache/arrow/driver/jdbc/accessor/impl/numeric/ArrowFlightJdbcDecimalVectorAccessor.java

```diff
@@ -71,6 +71,7 @@ public String getString() {
   }
 
   @Override
+  @SuppressWarnings("BigDecimalEquals")
```

Comment:
Use compareTo instead https://stackoverflow.com/questions/10950914/how-to-check-if-bigdecimal-variable-0-in-java